docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,598 @@
+.. include:: ../common/common_definitions.rst
+
+.. _pid_issuing.rst:
+
+PID Issuing
++++++++++++
+
+The relevant entities and interfaces involved in the issuing flows are:
+
+    - *Wallet Provider*: It represents an organization (public or private) that is responsible for the release of an eIDAS-compliant EUDI Wallet solution. It also issues to the Wallet Instance a Wallet Attestation by means of an Attestation Service. The Wallet Attestation certifies the genuinity and authenticity of the Wallet Instance and its compliance with a trust framework (e.g., with respect to the relevant security and privacy requirements).
+    - *Wallet Solution*: It represents the entire product and service owned by a Wallet Provider, offered to all Users of that solution. A Wallet Solution can be certified as being EUDI-compliant by a Conformity Assessment Body (CAB).
+    - *Wallet Instance*: the Wallet Solution is instantiated to Wallet Instance through installation and initialization on the User's device. it provides interfaces for User interaction with the Wallet Solution backend. 
+    - *PID Provider*: It represents the issuer of eIDAS Person Identification Data (PID). It is composed of:
+
+        - OIDC VCI Component: based on the “OpenID for Verifiable Credential Issuance” specification  `[OIDC4VCI. Draft 13] <https://openid.bitbucket.io/connect/openid-4-verifiable-credential-issuance-1_0.html>`_ to release PID credentials.
+        - National eID component (OIDC Core or SAML2): It represents the component to authenticate the End-User with the National IdPs.
+    - National IdP: It represents preexisting identity systems based on SAML2 or OIDC, already in production in each Member State (for Italy SPID and CIE id authentication schemed notified eIDAS with *LoA* **High**).
+    
+.. _fig_High-Level-Flow-EUDIW-PID-Issuing:
+.. figure:: ../../images/High-Level-Flow-EUDIW-PID-Issuing.svg
+    :figwidth: 100%
+    :align: center
+
+    PID Issuing - General architecture and high level flow
+
+The :numref:`fig_High-Level-Flow-EUDIW-PID-Issuing` shows a general architecture and highlights the main operations involved in the issuing of a PID, in particular:
+
+    0. **Wallet Instance Setup**: the first time the Wallet Instance is started a preliminary setup phase MUST be carried out. It consists of the release of a verifiable proof issued by the Attestation Service provided by the Wallet Provider that asserts the genuineness, the authenticity and the compliance with a trust framework of the Wallet Instance. The verifiable proof binds a public key corresponding to a local private key generated by the Wallet Instance. 
+    1. **Obtaining the trusted PID Provider**: the Wallet Instance queries the Trust Anchor to fetch the trusted PID Provider. 
```

Comment:
may we have a quick example of a request to the listing endpoint with entity_type set to openid_credential_issuer?